docker/dpcpp/Dockerfile

```diff
@@ -4,6 +4,9 @@ ARG IMPL_VERSION
 
 FROM ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:no-drivers-$IMPL_VERSION
 
+# Make sure that apt is executed with root previlegies
+USER root
```

Comment:
Why is this suddenly required?

Reply:
DPC++ recently changed the default user from `root` to unprivileged one. 
https://github.com/intel/llvm/pull/16411